### PR TITLE
RFC: Reduction techniques for regression

### PR DESCRIPTION
**Pull request by johannes9522**

### CODE REVIEW

**Comment by SteveBronder** [comment]
Small note, for groups you need to make the train/test cut on a day and not just a particular row, else if data has groups in it you can end up having group A at time point 10 in training and group B in time point 10 in testing which causes pollution

**Comment by johannes9522** [comment]
Thanks for your comments Steve! Especially the book you you mentioned seems very interesting. The issues about CV will be treated in more detail in the RFC about Resampling

**Comment by SteveBronder** [comment]
Wait sorry, what is this RFC about?

File: attic/rfc_reduction.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+ Feature Name: `Forecasting Reduction Techniques`
+ Start Date: 2019-11-16
+ Target Date:
+ ```
+
+## Summary
+[summary]: #summary
+ 
+ - Lagged values of a timeseries can be used as features for a machine learning algorithm.
+ - Two major apporaches: rolling window regression and expanding window regression
```
Comment (SteveBronder) [inline attic/rfc_reduction.md]: Sorry if we are not talking about CV then idt there is such a thing as rolling and expanding window regression. Those are methods for validating time series.

Tuning the amount of data you show a model seems, as a hyperparameter for performance, very weird and dangerous to me

File: attic/rfc_reduction.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+```
+
+
+
+## Rationale, drawbacks and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+These two approaches make it possible the reformulate time series forcasting as a regression problem.While training via the rolling window approach seems obvious, we have to come up with solutions for the expanding window approach, as most algorithms expect a fixed sized feature-dimensionality.
+
+Supposed we we want to predict the next h values of our series we have h regression problems, one for each timestep we want to predict. Through this approach the algorithm may not be capable to take advantages of the correlation between our targets. Another drawback is the hurt iid assumption that is usually required for machine learning algorithms.
```
Comment (SteveBronder) [inline attic/rfc_reduction.md]: I'm not really understanding this section

**Comment by SteveBronder** [comment]
> Thanks for your comments Steve!

Also np! Glad to see the ball rolling

File: attic/rfc_reduction.md (hunk 1 above)
Comment (johannes9522) [reply attic/rfc_reduction.md]: This RFC is about giving a basic idea of how to rephrase forecasting problems to ordianry regression problems. 
Sorry, I confused some things about the two methods. I will update the rfc soon.

**Comment by pfistfl** [comment]
johannes9522 <notifications@github.com> schrieb am Mo., 18. Nov. 2019,
17:59:

**Comment by pfistfl** [comment]
YySy-

johannes9522 <notifications@github.com> schrieb am Mo., 18. Nov. 2019,
17:59:

**Comment by pfistfl** [comment]
ÝÝ-)-Y-----ýýýÝýýýýR☆-yyyy

johannes9522 <notifications@github.com> schrieb am Mo., 18. Nov. 2019,
17:59:

**Comment by pfistfl** [comment]
Scary

johannes9522 <notifications@github.com> schrieb am Mo., 18. Nov. 2019,
17:59:

**Comment by berndbischl** [comment]
@pfistfl was genau machst du hier? sie garbage messages oben?

**Pull request closed without merging**